extensions/blocks/contact-form/components/jetpack-contact-form.js

```diff
@@ -190,6 +199,52 @@ class JetpackContactForm extends Component {
 		);
 	}
 
+	renderOnSubmissionFields() {
+		const {
+			customThankyou,
+			customThankyouType,
+			customThankyouMessage,
+			customThankyouRedirect,
+		} = this.props.attributes;
+		return (
+			<Fragment>
+				<ToggleControl
+					label={ __( 'Thank you message', 'jetpack' ) }
+					help={ __( 'Toggle to show a thank you message', 'jetpack' ) }
```

Comment:
The text of this seems a little redundant. Also, the [`ToggleControl` docs](https://developer.wordpress.org/block-editor/components/toggle-control/) shows the `help` text changing based on the state. @lessbloat, thoughts?

Reply:
You're right. Also, good eye on the ToggleControl help text changing. I decided to just remove the help text, and change the label to read: "Show a thank you message". Here's a preview: https://cl.ly/dae722da2330 I've updated all of the mockups within the [brief](https://docs.google.com/document/d/1xAUQyxBCVpa4gIeGWMf5dW6MVLBA_hhY15K89VJfN0E/edit#heading=h.mvgauik5kgjm) accordingly. 

Please let me know if anything else stands out!

Reply:
Thanks, I implemented that in 131fe50, but now it feels like an unnecessary extra click. Expanding the *On submission* panel seems like the "I want to use this option" action by the author, I'm not sure that they need to click another toggle.

Perhaps this toggle could be removed entirely, and the "Message type" dropdown changed "Submission Action"-type dropdown: the author can select from "Default behaviour", "Show a custom message", or "Redirect to another page".

Reply:
@pento I love that suggestion!

Here's a mockup:

![confirmation-options](https://user-images.githubusercontent.com/5634774/66793749-be99cb80-eecc-11e9-86cc-ffa312dc8b06.png)

Note: While we're in here, can we "Title Cap" `Email feedback settings`?

Reply:
Mockup looks good to me! Implemented in aa017992e (along with the `Email feedback settings` title-capping).